includes/title-meta.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Page Title Meta setup
+ *
+ * @package Go\TitleMeta
+ */
+
+namespace Go\TitleMeta;
```

Comment:
@AnthonyLedesma Lets update this class name to `Go\Title_Meta` just so we're matching the style in our other classes (`Title_Control`, `Switcher_Control`, `Range_Control` etc.). Also, I'm fairly certain core uses the same naming conventions.